guides/common/modules/proc_performing-post-upgrade-tasks-on-your-project-server.adoc

```diff
@@ -0,0 +1,19 @@
+[id="performing-post-upgrade-tasks-on-your-{project-context}-server_{context}"]
+= Performing post-upgrade tasks on your {ProjectServer}
+
+After upgrading your {ProjectServer}, perform the following tasks to ensure your environment is adjusted to any changes made during the upgrade.
+Some tasks might be optional depending on your specific configuration.
```

Comment:
```suggestion
Some steps depend on your specific {Project} configuration.
```

To me, there's a big difference between "optional" as is "I can if I want to" and "conditional" as in "If I have configured Foreman/Katello to do X, I have to do Y after  the upgrade".

Reply:
You're right. Applied, with a slightly different wording.